uploadservice/src/main/java/net/gotev/uploadservice/UploadTask.java

```diff
@@ -44,7 +44,7 @@
     /**
      * Contains all the parameters set in {@link UploadRequest}.
      */
-    protected UploadTaskParameters params = null;
+    protected UploadTaskParameters params = new UploadTaskParameters();
```

Comment:
@stephentuso I've left this to null here, because it always gets a value in the [init method](https://github.com/stephentuso/android-upload-service/blob/0a7e0808f0e3f051d28a5032d29f9eb191c26ec5/uploadservice/src/main/java/net/gotev/uploadservice/UploadTask.java#L123). Have you had some issues and then decided to create a new instance here?